GoogleApiWrappers/src/Google.Apis.Storage.v1.ClientWrapper/DownloadObjectOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright 2015 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Download;
+
+namespace Google.Apis.Storage.v1.ClientWrapper
+{
+    /// <summary>
+    /// Options for <c>DownloadObject</c> operations.
+    /// </summary>
+    public class DownloadObjectOptions
```

Comment:
That would be unconventional - we're following TAP:
https://msdn.microsoft.com/en-us/library/hh873175(v=vs.110).aspx